x-pack/platform/plugins/shared/alerting/public/pages/maintenance_windows/components/create_maintenance_windows_form.tsx

```diff
@@ -173,8 +167,8 @@ export const CreateMaintenanceWindowForm = React.memo<CreateMaintenanceWindowFor
           timezone: formData.timezone ? formData.timezone[0] : defaultTimezone,
           recurringSchedule: formData.recurringSchedule,
         }),
-        categoryIds: formData.solutionId ? [formData.solutionId] : null,
-        scopedQuery: availableSolutions.length !== 0 ? scopedQueryPayload : null,
+        scopedQuery: scopedQueryPayload ?? null,
+        ...(showMultipleSolutionsWarning || formData.solutionId ? { categoryIds: null } : {}),
```

Comment:
I think a safer solution would be:

```suggestion
        ...(scopedQueryPayload ? { categoryIds: null } : {}),
```

or

```suggestion
        ...(scopedQueryPayload || formData.solutionId ? { categoryIds: null } : {}),
```

Since we are removing `categoryIds` completely, `CreateMaintenanceWindowForm` should have sole responsibility over that.